rustler_codegen/src/nif.rs

```diff
@@ -48,15 +48,19 @@ pub fn transcoder_decorator(nif_attributes: NifAttributes, fun: syn::ItemFn) ->
     let argument_names = create_function_params(inputs.clone());
     let erl_func_name = nif_attributes
         .custom_name
-        .map(|n| syn::Ident::new(n.value().as_str(), Span::call_site()))
-        .unwrap_or_else(|| name.clone());
+        .map(|n| n.value().to_string())
```

Comment:
I think this could be fused with the unwrap below using `map_or_else()` (which does both rewrap with `Option`), but this here also works just fine 🙂